src/client/components/checkout/PaymentInfo.tsx

```diff
@@ -0,0 +1,76 @@
+import React from 'react'
+import { useParams } from 'react-router-dom'
+import styled from '@emotion/styled'
+import { colorsV3 } from '@hedviginsurance/brand'
+import { useCurrentLocale } from 'l10n/useCurrentLocale'
+import { useTextKeys } from 'utils/textKeys'
+import { getFormattedPrice } from 'utils/getFormattedPrice'
+import { usePriceQuery } from 'data/graphql'
+import { Spinner } from '../utils'
+
+const { gray900, gray700 } = colorsV3
+
+const Wrapper = styled.div`
+  display: grid;
+  grid-template-columns: auto;
+`
+const TotalPrice = styled.div`
+  font-size: 1rem;
+  line-height: 1.5rem;
+  color: ${gray900};
+`
+const CancelInfo = styled.div`
+  font-size: 0.75rem;
+  line-height: 1rem;
+  color: ${gray700};
+`
+const SpinnerWrapper = styled.div`
+  display: flex;
+  align-items: center;
+  font-size: 2rem;
+  color: ${gray900};
+`
+
+export const PaymentInfo = () => {
+  const textKeys = useTextKeys()
+  const { isoLocale, currencyLocale } = useCurrentLocale()
+  const { id: quoteCartId } = useParams<{ id: string }>()
+  const { data, loading: isLoading, error } = usePriceQuery({
+    variables: {
+      id: quoteCartId,
+      locale: isoLocale,
+    },
+  })
+  const quoteBundle = data?.quoteCart.bundle
+
+  if (error || !quoteBundle) {
+    return <div></div>
```

Comment:
Do we really want to return this or e.g. `null`?

Reply:
Oh, yeah, that's just to always keep the button on the right side of the footer, even if there's an error. But also doesn't make sense to render som error state since if this isn't working I'm pretty sure we'll have issues in other places as well and it makes more sense to render e.g. the error modal from the `CheckoutPageWrapper` for example. 
But I know this doesn't look good so note sure what's the best approach 🤔

Reply:
Hmm, could be a case where adding a "why" comment is warranted -- if there's no other simple way around it.